File: logica_de_programação/exercicios_em_C/atividade_estudoDirigidoC/exercicios_em_C/numeros_pares.c
#include <stdio.h>

int main() {
    int soma = 0;
    for (int i = 1; i <= 200; i++) {
        printf("%i -> ", i);
        if (i % 2 == 0) {
            soma += i;
        }
    }
    printf("FIM");
    printf("\nA soma total dos números pares é: %d\n", soma);
    return 0;
}

File: logica_de_programação/python/exercicios_em_python/numerospares.py
valores = []
pares = 0

for c in range(10):
    num = int(
        input('Digite um número: ')
    )
    valores.append(num)
    if num % 2 == 0:
        pares += 1
print(f'O total de números pares: {pares}')